packages/core/src/Markup.tsx

```diff
@@ -24,7 +24,7 @@ export default function Markup(props: MarkupProps) {
     mainContent = emptyContent;
   }
 
-  if (tag === 'fragment') {
+  if (!wrap) {
```

Comment:
Also breaking. Need to keep the fragment condition.

Reply:
Makes sense. Will fix.